lib/logger/utils.ts

```diff
@@ -49,6 +51,48 @@ const contentFields = [
   'yarnLockParsed',
 ];
 
+export default function prepareError(err: Error): Record<string, unknown> {
+  const response: Record<string, unknown> = {
+    ...err,
+  };
+
+  // Can maybe removed?
```

Comment:
Should we remove these now? I don't see how this helps anything

Reply:
it only helps, if the message or stack are non-enumerable properties on a custom error class. But i think we can remove them anyways. If we see something missin gin logs we can add handling on demand.

Reply:
OK, looks like this is still needed, because `message` and `stack` are not copied by default. 😢 So i'll revert the changes